static/app/views/releases/list/thresholdsDataContext.tsx

```diff
@@ -0,0 +1,26 @@
+import {createContext, ReactNode, useContext} from 'react';
+
+import useFetchThresholdsListData, {
+  EMPTY_THRESHOLDS_LIST_DATA,
+  HookProps,
+} from '../utils/useFetchThresholdsListData';
+
+interface ProviderProps extends HookProps {
+  children: ReactNode;
+}
+
+const ThresholdsListDataContext = createContext<
+  ReturnType<typeof useFetchThresholdsListData>
+>(EMPTY_THRESHOLDS_LIST_DATA);
+
+export function ThresholdsDataContext({children, ...listDataParams}: ProviderProps) {
```

Comment:
Context provider is not utilized atm, but will be useful for https://github.com/getsentry/sentry/pull/55886/files